main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os"
+
+	"github.com/Schrodinger-Hat/Daje/pkg/cmd/root"
```

Comment:
Why did you use the remote path for the package init and not the local relative path?